src/Microsoft.ML.ImageAnalytics/ExtensionsCatalog.cs

```diff
@@ -9,65 +9,117 @@ namespace Microsoft.ML
 {
     public static class ImageEstimatorsCatalog
     {
-        /// <summary>
-        /// Converts the images to grayscale.
-        /// </summary>
+        /// <include file='doc.xml' path='doc/members/member[@name="ImageGrayscalingEstimator"]/*' />
         /// <param name="catalog">The transform's catalog.</param>
-        /// <param name="columns">The name of the columns containing the image paths(first item of the tuple), and the name of the resulting output column (second item of the tuple).</param>
-        public static ImageGrayscalingEstimator ConvertToGrayscale(this TransformsCatalog catalog, params (string outputColumnName, string inputColumnName)[] columns)
-            => new ImageGrayscalingEstimator(CatalogUtils.GetEnvironment(catalog), columns);
+        /// <param name="columnPairs">The name of the columns containing the name of the resulting output column (first item of the tuple), and the paths of the images to work on (second item of the tuple).</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        ///  [!code-csharp[ConvertToGrayscale](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/ImageAnalytics/ConvertToGrayscale.cs)]
```

Comment:
So the file is called ConvertToGrayScale.cs - does casing matter? Should ConvertToGrayScale.cs be renamed?